docs/toolbox/guidelines/dockerfile-linting.md

```diff
@@ -54,6 +54,14 @@ In order to keep the feedback loop short and write best practice code straight o
 
 Having the CLI available means that we can manually run linting, or even better, that we can attach it to a pre-commit hook so that it runs automatically without us having to think about it. Connecting it to a local **build** process is also a good idea.
 
+##### In Docker
```

Comment:
Thanx, this makes sense. Would i be valuable to add an alternative code section showing how to use the command line params with hadolint to fine tune it's behavior? Something like this

`cat Dockerfile | docker run --rm -i hadolint/hadolint:latest hadolint --failure-threshold=error -`

And then some text explaining that the exit code from running the container could be used to drive the further logic of the CI process?